src/Sylius/Behat/Page/Shop/Product/ShowPageInterface.php

```diff
@@ -54,6 +54,8 @@ public function getName(): string;
 
     public function getPrice(): string;
 
+    public function getOriginalPrice(): string;
+
```

Comment:
this is a BC Break.

Reply:
Behats are not covered with our BC promise ;)